views/partials/_mobile-navigation.njk

```diff
@@ -69,7 +69,7 @@
   </ul>
   {# No JS fallback to aid with navigation #}
   {# To do: Implement logic. Don't show on homepage #}
-  <div class="app-c-mobile-nav-breadcrumbs">
+  <div class="app-c-mobile-nav__breadcrumbs">
     {{- govukBreadcrumb({
```

Comment:
actually could we just pass this class to the breadcrumb component though classes parameter?

Reply:
Nice one @igloosi 💯  Amended.